dashboard/test/lib/quick_assign_helper.rb

```diff
@@ -68,9 +124,9 @@ class QuickAssignHelperTest < ActionController::TestCase
     grouped_offerings = QuickAssignHelper.group_grade_level_offerings([course_offering1, course_offering2, course_offering3, course_offering4, course_offering5], teacher, I18n.default_locale)
 
     assert_equal 1, grouped_offerings['Course']['Header 2'].length
-    assert_equal course_offering1.id, grouped_offerings['Course']['Header 1'][0][:id]
+    assert_equal course_offering2.id, grouped_offerings['Course']['Header 1'][0][:id]
```

Comment:
Just curious about what's going on here

Reply:
This was weird. This test started to fail and I'm not sure why. I meant to look into it more then forgot -- I'm a little worried this test isn't being run in Drone for some reason